lib/host/src/keystone.cpp

```diff
@@ -118,42 +282,36 @@ keystone_status_t Keystone::destroy()
 
 #define KEYSTONE_ENCLAVE_EDGE_CALL_HOST  11
```

Comment:
Pretty sure I'm the one that left this here, we should move it out. Optional on this PR.